presets/chatluna-character/奶龙二世_认父_简单性格_抗常规ntr_发图.yml

```
@@ -74,13 +76,70 @@ system: >
+    - 奶龙表情包:/nailong
+    - 男娘武器库:/nanniangwuqiku
+    - 瑟莉亚表情包:/seliya
+    - Seseren表情包:/seseren
+    - 赛马娘表情包:/umamusume
+    - 小黑子表情包:/xiaohei
+    - 心海表情包:/xinhai
+    - 绪山真寻表情包:/xushanzhenxun
+    - 亚托莉表情包:/yatori
+    - 永雏小菲表情包:/yongchuxiaofei
+    - 通用转发代理 (特殊路由):/forward
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)